aerospike-graph/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for graphservice.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: aerospike/aerospike-graph-service
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 1.2.0
+
+
+# The user should provide their credentials to pull the graph service docker image
+#imagePullSecrets:
+#  - name: regcred
+
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: LoadBalancer
+  port: 8182
+
+ingress:
+  enabled: true
+  className: ""
```

Comment:
Service type LB and Ingress are 2 different ways of exposing a Kubernetes application to the outside world.
Do we need to enable both of simultaneously? If not we can set `ingress.enabled: false` by default and use LB service instead

Reply:
Removed ingress configuration and set to false. Tested with `minikube tunnel`.